lang/java/build.sh

```diff
@@ -34,6 +34,16 @@ main() {
         mvn -B test
         # Test the modules that depend on hadoop using Hadoop 2
         mvn -B test -Phadoop2
+        # Test external avro archetype generation
+        mvn archetype:generate \
+          -DarchetypeGroupId=org.apache.avro \
+          -DarchetypeArtifactId=avro-service-archetype \
+          -DgroupId=org.apache.avro \
```

Comment:
How does it know which version of the archetype to use ?
I expect something like `-DarchetypeArtifactId=1.12.0-SNAPSHOT` to be specified too.

Reply:
I did not want to do that to avoid having to update the current version on every release (usually project maintainers tend to forget that). I digged deeper and found that there is a way to do integration tests on maven archetypes so it is maybe a better solution (Avro already was doing it but it was not well configurated for modern maven standards).
https://maven.apache.org/archetype/maven-archetype-plugin/integration-test-mojo.html

I will update this in the PR